### PR TITLE
DB-2337 add syslog support

### PR DESCRIPTION
**Pull request by sincejune**

### CODE REVIEW

File: dashbase/values.yaml [hunk 1]
```diff
@@ -522,6 +522,18 @@ services:
 
 
 #### Optional services ###
+  fluentd:
```
Comment (javasoze) [inline dashbase/values.yaml:525]: Let's not call it fluentd, call it says log?

Comment (sincejune) [reply dashbase/values.yaml]: renamed to `syslog`

**Comment by sincejune** [comment]
Our chart will use `dashbase/<service_name>:nightly` as the default image of service. Name this service `syslog` will let it pull `dashbase/syslog:nightly` by default.
I added the `componentName` property to avoid the issue, but it could be the blocker when we need to change `fluentd` to something else.

**Comment by sincejune** [comment]
We need to disable `syslog` and re-enable to switch `componentName` from `fluentd` to others.
```
➜  ~ helm upgrade dashbase ~/dashbase/dashbase-k8s/dashbase -f dashbase.yml -i
Error: UPGRADE FAILED: cannot patch "syslog" with kind Deployment: Deployment.apps "syslog" is invalid: spec.selector: Invalid value: v1.LabelSelector{MatchLabels:map[string]string{"app":"dashbase", "component":"filebeat"}, MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: field is immutable
```

**Pull request merged**